tox.ini

```diff
@@ -12,6 +12,10 @@ extend-ignore=E203
 max-line-length=88
 per-file-ignores=__init__.py:F401
 
+[blocklint]
+max_issue_threshold=1
+blocklist=blacklist,whitelist,master,slave,owner,master,native,culture,housekeeping
```

Comment:
I can be swayed on what terms we want to check. Notably, `master` and `native` appears to pop up in some URLs. 

Placing `master` under the `exactlist` configuration seems to get around that pretty well. I could add that in here if necessary. Otherwise I left it open ended because each project could differ on a case by case basis.

Reply:
`blacklist`, `whitelist`, `master`, and `slave` I understand. Can use the `exactlist` per lib for `master` as you did in the `web3` implementation. The others I'm not against, but don't understand why they should be excluded. `owner` would show up if we decide to use the github codeowners functionality, for example.